apps/meteor/app/lib/server/lib/notifyListener.ts

```diff
@@ -87,6 +89,18 @@ export const notifyOnRoomChangedByUsernamesOrUids = withDbWatcherCheck(
 	},
 );
 
+export const notifyOnRoomChangedByContactId = withDbWatcherCheck(
+	async <T extends IOmnichannelRoom>(
+		contactId: Exclude<T['v']['contactId'], undefined>,
+		clientAction: ClientAction = 'updated',
+	): Promise<void> => {
+		const items = LivechatRooms.findByContactId(contactId);
```

Comment:
Should this be open rooms only?

Reply:
Hmmm right, that makes more sense. Changed it